src/style.css

```
@@ -0,0 +1,79 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir" ;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
```

```suggestion
  font-weight: 300;
```
bc you use `300` on 27 line